scenarios/glue_privesc/terraform/ec2.tf

```diff
@@ -0,0 +1,83 @@
+resource "aws_key_pair" "bob-ec2-key-pair" {
```

Comment:
## General for several resources ([linting](https://github.com/terraform-linters/tflint))

Interpolation-only expressions are deprecated in Terraform v0.12.14
> "${var.scenario-name}" --> var.scenario-name

Run `terraform fmt` there are a few variables that